test/unit/app/util/format-versions.spec.js

```diff
@@ -0,0 +1,46 @@
+/* eslint no-use-before-define: off */
+import chalk from 'chalk';
+
+import formatVersions from '../../../../app/util/format-versions';
+
+describe('formatVersions', () => {
+  it('should handle versions being undefined correctly', () => {
+    expect(formatVersions(undefined)).toBe('');
+  });
+
+  it('should handle versions.version being undefined correctly', () => {
+    expect(formatVersions({ versions: undefined })).toBe('');
+  });
+
+  it('should handle versions.version being empty array', () => {
+    expect(formatVersions({ versions: [] })).toBe('');
+  });
+
+  it('should handle versions.version being set with correct values', () => {
+    const expected = `${chalk.yellow(chalk.underline('Version skews'))}
+
+webpack:
+  a:
+      - Abc
+`;
+
+    expect(formatVersions(correctVersionsFixture())).toEqual(expected);
+  });
+});
+
+const Problem = {
```

Comment:
Hi @maoueh I would prefer you re-enabled the rule and move the declarations to the top. I actually find this more readable and consistent with the rest of the code base. The fix looks good to me so will merge when you've done this.